### PR TITLE
feat: add Template and Asset chooser dummies

### PR DESCRIPTION
**Pull request by utmishra**
**Provides dummy values to tests like `getTemplateById`, `openAssetChooser`, etc.**
- Needed for [#14704](https://github.com/Frontify/clarify/pull/14704)

### CODE REVIEW

File: packages/app-bridge/src/types/AssetChooser.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AssetChooserObjectType } from './Terrific';
+
+export type AssetChooserFilter = {
```
Comment (SamuelAlev) [inline packages/app-bridge/src/types/AssetChooser.ts:5]: Please use the existing types, we also don't allow every filters, you can find the one we allow in `AssetChooserOptions`

Comment (utmishra) [reply packages/app-bridge/src/types/AssetChooser.ts:5]: Hey, I can remove this file and update `AssetChooserOptions` (in [`types/Terrific.ts`](https://github.com/Frontify/brand-sdk/blob/e948fb0b49cc638d0aa6e61b23ab5a123ebce36d/packages/app-bridge/src/types/Terrific.ts#L44)) type to look like this (to enforce the allowed filters)
```
export type AssetChooserOptions = {
    // ... existing fields ...
    filters?: {
        key: 'id' | 'ext' | 'object_type' | 'external_url';
        values?: (number | string)[];
        inverted?: boolean;
        containsText?: string;
    }[];
} & (
    // ... selectedValueId or selectedValueIds
);
```

Would that work?

Comment (utmishra) [reply packages/app-bridge/src/types/AssetChooser.ts]: Done!

File: packages/app-bridge/src/tests/AssetChooserOptionsDummy.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AssetChooserObjectType, AssetChooserOptions, AssetChooserProjectType, FileExtension } from '../types';
+
+export class AssetChooserOptionsDummy {
+    static default(): { brandId: number } & AssetChooserOptions {
```
Comment (SamuelAlev) [inline packages/app-bridge/src/tests/AssetChooserOptionsDummy.ts:6]:
```suggestion
    static default(): AssetChooserOptions {
```

File: packages/app-bridge/src/types/Terrific.ts [hunk 3]
```diff
@@ -47,6 +47,12 @@ export type AssetChooserOptions = {
     multiSelection?: boolean;
     extensions?: (FileExtension | string)[];
     urlContains?: string;
+    filters?: {
+        key: 'id' | 'ext' | 'object_type' | 'external_url';
+        values?: (number | string)[];
+        inverted?: boolean;
+        containsText?: string;
+    }[];
```
Comment (SamuelAlev) [inline packages/app-bridge/src/types/Terrific.ts:55]:
```suggestion
```

**Review by SamuelAlev — CHANGES_REQUESTED** [review]

File: packages/app-bridge/src/types/Terrific.ts (hunk 3 above)
Comment (utmishra) [reply packages/app-bridge/src/types/Terrific.ts]: Removed

File: packages/app-bridge/src/tests/AssetChooserOptionsDummy.ts (hunk 2 above)
Comment (utmishra) [reply packages/app-bridge/src/tests/AssetChooserOptionsDummy.ts]: Removed

**Review by SamuelAlev — APPROVED** [review]
Looks great!

**Pull request merged**